### PR TITLE
Add form styling + show file uploads

### PR DESCRIPTION
**Pull request by dhaselhan**
* Add new columns to store file name and file size
* Show files attached to the application on the file picker
* Update file picker to have mobile styles
* Add form controls scss file to clean up our form styling
* Add GRPC calls for getting an open URL and deleting documents

### CODE REVIEW

File: portal-frontend/src/app/shared/file-drag-drop/file-drag-drop.component.scss [hunk 1]
```diff
@@ -28,3 +40,39 @@
   align-items: center;
   padding: 2rem;
 }
+
+.uploaded-file {
+  border: 1px solid #000000;
+  border-radius: 4px;
+  padding: 8px;
```
[the remark this answers is not in the document]
Comment (dhaselhan) [reply portal-frontend/src/app/shared/file-drag-drop/file-drag-drop.component.scss:47]: ah crap! I copied it from figma and forgot to update

[the remark this answers is not in the document]
Comment (dhaselhan) [reply portal-frontend/src/app/shared/file-drag-drop/file-drag-drop.component.scss]: fixed!

File: portal-frontend/src/app/shared/file-drag-drop/file-drag-drop.component.scss [hunk 2]
```diff
@@ -28,3 +40,39 @@
   align-items: center;
   padding: 2rem;
 }
+
+.uploaded-file {
+  border: 1px solid #000000;
```
[the remark this answers is not in the document]
Comment (dhaselhan) [reply portal-frontend/src/app/shared/file-drag-drop/file-drag-drop.component.scss]: Changed to $black

**Review by mhuseinov — APPROVED** [review]
💯

**Pull request merged**